pkg/controller/migmigration/task.go

```diff
@@ -572,8 +614,8 @@ func (t *Task) init() {
 	}
 }
 
-// Advance the task to the next phase.
-func (t *Task) next() {
+// Set the step which will be executed.
+func (t *Task) setStep() int {
```

Comment:
Please explain how this change is related to and required for this feature.  Else, revert.